src/module.c

```diff
@@ -2205,6 +2205,14 @@ int RM_ReplyWithDouble(RedisModuleCtx *ctx, double d) {
     return REDISMODULE_OK;
 }
 
+/* Send a BigNumber reply */
```

Comment:
the doc comment feels a bit lean, maybe extend it a bit to explain what's a big number?
or state that in RESP3 it'll just be a string?
i think we have other APIs with similar comments.

Reply:
added a comment and a question

Reply:
I think it's ok that we don't do any verification, we should make it clear that from redis's perspective it's just a string that is forwarded to the client with an indication that it denotes a long number (in RESP3).
and in RESP2 it's just a string.
This should make it clear that there's no verification in redis 8-)

There are a few other examples of APIs doing a similar thing see `RM_ReplyWithMap` and `RM_ReplyWithNullArray`.
i.e. explain what the API does in RESP2.

I think we should also use this opportunity to improve the doc of `RM_ReplyWithBool` to also denote that in RESP2 it's replying with 0 or 1.

Reply:
so reading the RM_ReplyWithDouble it already makes it clear that its a string, even in RESP3.  I added this line

 * In RESP3 the string is tagged as a double, while in RESP2 its just a plain string 
 * that the user will have to parse.